src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/trial_of_the_champion.cpp

```diff
@@ -33,28 +33,227 @@ EndContentData */
 #include "Vehicle.h"
 #include "Player.h"
 
-enum Yells
+#define GOSSIP_START_EVENT1     "I am ready."
```

Comment:
The text `'I am ready.'` exists in `broadcast_text`, but not in `npc_text` or `creature_text` yet.
```sql
SELECT * FROM `broadcast_text` WHERE `MaleText`= 'I am ready.' OR `FemaleText`= 'I am ready.';